templates/agent/zabbix-agentd.d/couchbase.conf.j2

```diff
@@ -0,0 +1,3 @@
+# {{ ansible_managed }}
+
+UserParameter=couchbase[*], python {{ zabbix_prefix_dir }}/scripts/couchbase.py "$1" "$2" "$3" "$4"
```

Comment:
Related to the `discovery` items mentioned above (cluster / bucket) - you may want to use something closer to https://github.com/wiredcraft-ops/role-wcl-zabbix-scripts/blob/master/templates/agent/zabbix-agentd.d/haproxy.conf.j2 - rather than the current discovery items:

- couchbase[bucket,names,string_list]
- couchbase[cluster,names,string_list]

Instead you could create something like:

- couchbase.discovery[clusters]
- couchbase.discovery[buckets]

And maybe even join them into a single discovery item...